app/uk/gov/hmrc/economiccrimelevyregistration/forms/contacts/SecondContactEmailFormProvider.scala

```diff
@@ -23,7 +23,7 @@ import javax.inject.Inject
 
 class SecondContactEmailFormProvider @Inject() extends Mappings {
 
-  private val maxLength = 160
+  private val maxLength = 132
```

Comment:
Should this be referencing uk.gov.hmrc.economiccrimelevyregistration.forms.mappings.MaxLengths.EmailMaxLength instead?